pages/common/chafa.md

```diff
@@ -0,0 +1,25 @@
+# chafa
+
+> Image printing in the terminal.
+> See also: `catimg` and `pixterm`.
+> More information: <https://hpjansson.org/chafa>.
+
+- Print an image or animated GIF to the terminal:
+
+`chafa {{path/to/input}}`
+
+- Use 24-bit [c]olor:
```

Comment:
```suggestion
- Print an image using 24-bit [c]olor:
```

Reply:
I think it gets a little dry to say "print" over and over. I already specified that it prints to the terminal in the first example. And since its primary function is to generate images and print them to the terminal, I think "use" should be good enough.

Reply:
The example descriptions shouldn't be described using the context. Make them independent

Reply:
Hm. I was under the impression that the format is useful for both showing independent examples and giving a brief tutorial as a whole. This is even somewhat supported by the guides, where it says to start with simple examples before going to more complex ones. Maybe this particular point isn't a good example of that, but is it really better to use the same verbage over and over, when the primary use of the tool has already been demonstrated?